### PR TITLE
Added a readme for Module 4 IDV webapp

### PR DESCRIPTION
**Pull request by vineetkachhawaha**
*Issue #, if available:*

*Added a readme for Module 4 IDV webapp*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: identity-verification/module_4/README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+- Register a new user after face match against an ID card with ID card data extraction
+- Authenticate returning user
+
+## Solution Deployment
+
+---
+
+
+- <a href="https://console.aws.amazon.com/cloudformation/home?region=us-east-1#/stacks/create/template?stackName=Riv-Prod&amp;templateURL=https://aws-ml-blog.s3.amazonaws.com/artifacts/rekognition-identity-verification-solution/Riv-Prod.template.json" target="_blank" rel="noopener noreferrer"><img class="alignnone wp-image-37966 size-full" src="https://d2908q01vomqb2.cloudfront.net/f1f836cb4ea6efb2a0b1b99f41ad8b103eff4b59/2022/06/14/ML-8625-image006-launch-stack.png" alt="Launch stack button" width="107" height="20"></a>&nbsp; N. Virginia (<code>us-east-1</code>)
+- Invoke the web UI - The web portal is deployed with Amplify. On the Amplify console, locate the hosted web application environment (Riv-Prod) and the URL. Copy the URL and access it from your browser.
```
Comment (matteofigus) [inline identity-verification/module_4/README.md:17]: I deployed the stack but I see nothing in the Amplify console for that region. 
1. Can you verify the deployment to test correct Amplify setup? 
2. Can we have the Amplify URL part of the Stack Output to simplify?
3. Why is the stack creating subnets and NAT gateway? Do we need that for a simple serverless app?

**Review by matteofigus — CHANGES_REQUESTED** [review]

File: identity-verification/module_4/README.md (hunk 1 above)
Comment (vineetkachhawaha) [reply identity-verification/module_4/README.md:17]: Thanks Matteo. 
for #3 The backend code was already developed , we extended the code to add the FE functionality. 
#1 ad #2 Deployment does check the correct Amplify setup. and I was not able to get the amplify deploy URL in the AWS CDK. 
And are you getting error in the Stack or does it deploy successfully?

Comment (matteofigus) [reply identity-verification/module_4/README.md:17]: On the EC2/VPC stuff, can you still remove what you don't need? I see EIP and NATs there, that's lots of extra cost that I would prefer to remove if don't needed.

For the deployment, I have no errors, but there's nothing about Amplify being deployed. If I go to Amplify, I see nothing and even on resources, nothing. Here's my list of resource types deployed:

```sh
❯ aws cloudformation describe-stack-resources --stack-name Riv-Prod --region us-east-1 | jq -r '.StackResources[].ResourceType'

AWS::Lambda::Function
AWS::IAM::Role
AWS::Lambda::Function
AWS::IAM::Role
AWS::IAM::Policy
AWS::CDK::Metadata
Custom::AWS
AWS::IAM::Policy
AWS::SSM::Parameter
AWS::EC2::SecurityGroup
AWS::EC2::VPCEndpoint
AWS::EC2::VPCEndpoint
AWS::EC2::VPC
AWS::EC2::Route
AWS::EC2::SubnetRouteTableAssociation
AWS::EC2::RouteTable
AWS::EC2::Subnet
AWS::EC2::Route
AWS::EC2::RouteTable
AWS::EC2::SubnetRouteTableAssociation
AWS::EC2::Subnet
AWS::EC2::InternetGateway
AWS::EC2::Route
AWS::EC2::EIP
AWS::EC2::NatGateway
AWS::EC2::RouteTable
AWS::EC2::SubnetRouteTableAssociation
AWS::EC2::Subnet
AWS::EC2::Route
AWS::EC2::SubnetRouteTableAssociation
AWS::EC2::RouteTable
AWS::EC2::Subnet
AWS::EC2::SubnetRouteTableAssociation
AWS::EC2::RouteTable
AWS::EC2::Subnet
AWS::EC2::SubnetRouteTableAssociation
AWS::EC2::RouteTable
AWS::EC2::Subnet
AWS::EC2::VPCGatewayAttachment
AWS::EC2::SecurityGroup
AWS::S3::Bucket
AWS::S3::Bucket
Custom::S3BucketNotifications
AWS::S3::BucketPolicy
AWS::SNS::Topic
AWS::SQS::Queue
AWS::SQS::QueuePolicy
AWS::SNS::Subscription
AWS::SNS::TopicPolicy
AWS::DynamoDB::Table
AWS::Logs::LogGroup
AWS::StepFunctions::StateMachine
AWS::IAM::Role
AWS::IAM::Policy
AWS::Logs::LogGroup
AWS::StepFunctions::StateMachine
AWS::IAM::Role
AWS::IAM::Policy
AWS::Logs::LogGroup
AWS::StepFunctions::StateMachine
AWS::IAM::Role
AWS::IAM::Policy
AWS::Logs::LogGroup
AWS::StepFunctions::StateMachine
AWS::IAM::Role
AWS::IAM::Policy
AWS::Lambda::Function
AWS::IAM::Role
AWS::IAM::Policy
AWS::Lambda::LayerVersion
AWS::Lambda::Function
AWS::IAM::Role
AWS::IAM::Policy
AWS::Lambda::LayerVersion
AWS::Lambda::Function
AWS::IAM::Role
AWS::IAM::Policy
AWS::Lambda::LayerVersion
AWS::Lambda::Function
AWS::IAM::Role
AWS::IAM::Policy
AWS::Lambda::LayerVersion
AWS::Lambda::Function
AWS::IAM::Role
AWS::IAM::Policy
AWS::Lambda::LayerVersion
AWS::Lambda::Function
AWS::IAM::Role
AWS::IAM::Policy
AWS::Lambda::LayerVersion
AWS::SSM::Parameter
AWS::ApiGateway::Model
AWS::ApiGateway::Model
AWS::ApiGateway::Model
AWS::ApiGateway::Model
AWS::ApiGateway::Model
AWS::ApiGateway::Model
AWS::ApiGateway::Model
AWS::IAM::Role
AWS::ApiGateway::RestApi
```

Is this code somewhere public so I can have a look?

Comment (vineetkachhawaha) [reply identity-verification/module_4/README.md:17]: https://github.com/aws-samples/rekognition-identity-verification

Comment (vineetkachhawaha) [reply identity-verification/module_4/README.md:17]: Thanks Matteo for catching this. I noticed the template was not updated . I have updated the template. Please try to deploy the stack again.

Comment (matteofigus) [reply identity-verification/module_4/README.md:17]: Hi Vineet, I deployed a new stack and now I get the frontend and the link is in the Stack Output, so my concern 1 and 2 are mitigated.

I still am concerned about those components that are not needed. I see VPC endpoints, EIP, NAT gateways, can you confirm all of that is not needed?

Can we please at least add here an alert to clarify that the customer will incur in XX$ costs to run the demo, and they should go to CloudFormation and delete the stack as soon as they're done?

Obviously, my preference would still be to remove the non-serverless components from the stack, which would remove the cost concern but if we can't, at least let's be ultra transparent. What do you think?

Comment (vineetkachhawaha) [reply identity-verification/module_4/README.md:17]: Hi Matteo, We have added the wording in the cleanup section -https://catalog.us-east-1.prod.workshops.aws/workshops/4a2a9a24-071d-4d96-b9be-0cc57b7db434/en-US/usecases/identity-verification/module-cleanup

Comment (matteofigus) [reply identity-verification/module_4/README.md:17]: Ok you convinced me. But please if you could prioritise the improvements to reduce the non-needed components that would be best. I'll add a second team mate for a review now, after the second approval we can merge.

**Review by matteofigus — APPROVED** [review]

**Review by ctd — APPROVED** [review]

**Pull request merged**